bakery/src/scripts/link_single.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
 exec > >(tee "${COMMON_LOG_DIR}"/log >&2) 2>&1
 
-if [[ -n "${TARGET_BOOK}" ]]; then
-    cp "${BAKED_INPUT}/${TARGET_BOOK}.baked.xhtml" "${LINKED_OUTPUT}/${TARGET_BOOK}.linked.xhtml"
-else
-    link-single "$BAKED_INPUT" "$BAKED_META_INPUT" "$(cat "${BOOK_INPUT}"/slug)" "$LINKED_OUTPUT/$(cat "$BOOK_INPUT"/slug).linked.xhtml"
-fi
+link-single "$BAKED_INPUT" "$BAKED_META_INPUT" "$(cat "${BOOK_INPUT}"/slug)" "$LINKED_OUTPUT/$(cat "$BOOK_INPUT"/slug).linked.xhtml" "'$TARGET_BOOK'"
```

Comment:
Is the use of the single quotes around `$TARGET_BOOK` purposeful here? I think even if `$TARGET_BOOK` is empty, this will get passed as `""` to the script w/o that, no?

Reply:
I'll double-check, but it appeared to be dropping it. I was getting not enough values to unpack.